packages/ui/BudgetIcon/index.tsx

```diff
@@ -0,0 +1,35 @@
+import styled from "styled-components";
+import { ReactNode } from "react";
+import { Icon, IconType } from "../Icon";
+
+type BudgetIconProps = {
+  icon?: IconType;
+  children: ReactNode;
+};
```

Comment:
```suggestion
type BudgetIconProps = {
  icon?: IconType;
} & React.HTMLProps<HTMLDivElement>;
```
TS will not shout at this 😄

Reply:
Fixed.